Description: replace ttf-kochi by fonts-ipafont
 ttf-kochi-{mincho,gothic} replaced by fonts-ipafont-{mincho,gothic}
Author: HIGUCHI Daisuke (VDR dai) <dai@debian.org>
Index: chromium-browser-29.0.1547.57/src/build/install-build-deps.sh
===================================================================
--- chromium-browser-29.0.1547.57.orig/src/build/install-build-deps.sh	2013-08-25 03:11:06.000000000 +0900
+++ chromium-browser-29.0.1547.57/src/build/install-build-deps.sh	2013-10-08 22:52:05.000000000 +0900
@@ -88,7 +88,7 @@
           libxslt1-dev libxss-dev libxt-dev libxtst-dev mesa-common-dev
           metacity patch perl php5-cgi pkg-config python python-cherrypy3
           python-dev python-psutil rpm ruby subversion ttf-dejavu-core
-          ttf-indic-fonts ttf-kochi-gothic ttf-kochi-mincho ttf-thai-tlwg
+          ttf-indic-fonts fonts-ipafont-gothic fonts-ipafont-mincho ttf-thai-tlwg
           wdiff git-core
           $chromeos_dev_list"
 
Index: chromium-browser-29.0.1547.57/src/content/shell/webkit_test_platform_support_linux.cc
===================================================================
--- chromium-browser-29.0.1547.57.orig/src/content/shell/webkit_test_platform_support_linux.cc	2013-08-25 03:11:45.000000000 +0900
+++ chromium-browser-29.0.1547.57/src/content/shell/webkit_test_platform_support_linux.cc	2013-10-08 22:52:05.000000000 +0900
@@ -39,8 +39,8 @@
 }
 
 const char* const kFonts[] = {
-    "/usr/share/fonts/truetype/kochi/kochi-gothic.ttf",
-    "/usr/share/fonts/truetype/kochi/kochi-mincho.ttf",
+    "/usr/share/fonts/opentype/ipafont-gothic/ipag.ttf",
+    "/usr/share/fonts/opentype/ipafont-mincho/ipam.ttf",
     "/usr/share/fonts/truetype/msttcorefonts/Arial.ttf",
     "/usr/share/fonts/truetype/msttcorefonts/Arial_Bold.ttf",
     "/usr/share/fonts/truetype/msttcorefonts/Arial_Bold_Italic.ttf",
Index: chromium-browser-29.0.1547.57/src/third_party/WebKit/Tools/DumpRenderTree/chromium/android_fallback_fonts.xml
===================================================================
--- chromium-browser-29.0.1547.57.orig/src/third_party/WebKit/Tools/DumpRenderTree/chromium/android_fallback_fonts.xml	2013-08-25 03:25:39.000000000 +0900
+++ chromium-browser-29.0.1547.57/src/third_party/WebKit/Tools/DumpRenderTree/chromium/android_fallback_fonts.xml	2013-10-08 22:52:05.000000000 +0900
@@ -7,7 +7,7 @@
     </family>
     <family>
         <fileset>
-            <file>kochi-mincho.ttf</file>
+            <file>ipam.ttf</file>
         </fileset>
     </family>
     <family>
Index: chromium-browser-29.0.1547.57/src/third_party/WebKit/Tools/Scripts/webkitpy/layout_tests/port/chromium_android.py
===================================================================
--- chromium-browser-29.0.1547.57.orig/src/third_party/WebKit/Tools/Scripts/webkitpy/layout_tests/port/chromium_android.py	2013-08-25 03:25:40.000000000 +0900
+++ chromium-browser-29.0.1547.57/src/third_party/WebKit/Tools/Scripts/webkitpy/layout_tests/port/chromium_android.py	2013-10-08 22:52:05.000000000 +0900
@@ -110,7 +110,7 @@
     [['/usr/share/doc/ttf-mscorefonts-installer/'], 'READ_ME!.gz', MS_TRUETYPE_FONTS_PACKAGE],
     # Other fonts: Arabic, CJK, Indic, Thai, etc.
     [['/usr/share/fonts/truetype/ttf-dejavu/'], 'DejaVuSans.ttf', 'ttf-dejavu'],
-    [['/usr/share/fonts/truetype/kochi/'], 'kochi-mincho.ttf', 'ttf-kochi-mincho'],
+    [['/usr/share/fonts/opentype/ipafont-mincho/'], 'ipam.ttf', 'fonts-ipafont-mincho'],
     [['/usr/share/fonts/truetype/ttf-indic-fonts-core/'], 'lohit_hi.ttf', 'ttf-indic-fonts-core'],
     [['/usr/share/fonts/truetype/ttf-indic-fonts-core/'], 'lohit_ta.ttf', 'ttf-indic-fonts-core'],
     [['/usr/share/fonts/truetype/ttf-indic-fonts-core/'], 'MuktiNarrow.ttf', 'ttf-indic-fonts-core'],
Index: chromium-browser-29.0.1547.57/src/ui/base/strings/app_locale_settings_ja.xtb
===================================================================
--- chromium-browser-29.0.1547.57.orig/src/ui/base/strings/app_locale_settings_ja.xtb	2013-08-25 03:13:35.000000000 +0900
+++ chromium-browser-29.0.1547.57/src/ui/base/strings/app_locale_settings_ja.xtb	2013-10-08 22:54:38.512742914 +0900
@@ -11,7 +11,7 @@
   <translation id="IDS_WEB_FONT_FAMILY">Helvetica,Hiragino Kaku Gothic Pro,sans-serif</translation>
 </if>
 <if expr="is_linux and not pp_ifdef('chromeos')">
-  <translation id="IDS_WEB_FONT_FAMILY">VL PGothic,Sazanami Gothic,Kochi Gothic,sans-serif</translation>
+  <translation id="IDS_WEB_FONT_FAMILY">IPAPGothic,VL PGothic,sans-serif</translation>
 </if>
 <if expr="pp_ifdef('chromeos') and pp_ifdef('_google_chrome')">
   <translation id="IDS_WEB_FONT_FAMILY">MotoyaG04Gothic, Noto Sans UI, sans-serif</translation>
